pkg/controller/cloudmetrics/cloudmetrics_controller.go

```diff
@@ -32,6 +32,7 @@ const (
 	redisMemoryUsagePercentageAverage = "cro_redis_memory_usage_percentage_average"
 	redisFreeableMemoryAverage        = "cro_redis_freeable_memory_average"
 	redisCPUUtilizationAverage        = "cro_redis_cpu_utilization_average"
+	redisEngineCPUUtilizationAverage        = "cro_redis_engine_cpu_utilization_average"
```

Comment:
Perhaps the alignment needs to be formatted here. At least in my editor this happens automatically.

Reply:
make code/fix would resolve it too. 🥳